dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -1423,18 +1416,30 @@ void Segment::placeDeltaIndex(DMContext & dm_context)
                 {RowKeyRange::newAll(is_common_handle, rowkey_column_size)});
 }
 
+String Segment::logId() const
+{
+    return fmt::format("<segment_id={} @{}>",
```

Comment:
"<segment_id=123 @3>" seems not easy to understand. Maybe explicitly show "<segment_id=123 epoch=3>"?